File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.py
class Solution(object):
    def differenceOfSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        elements=0
        digits=0
        for i in nums:
            elements+=i
            while i>0:
                digits+=i%10
                i=i//10
        res=elements-digits
        if res<0:
            res=0-res
        return res

File: 2535-difference-between-element-sum-and-digit-sum-of-an-array/2535-difference-between-element-sum-and-digit-sum-of-an-array.cpp
class Solution {
public:
    int differenceOfSum(vector<int>& nums) {
        int element=0;
        int digit=0;
        for(int i : nums){
            element+=i;
            while(i>0){
                digit+=i%10;
                i=i/10;
            }
        }
        return abs(element-digit);
    }
};